src/components/dialog/dialog.spec.js

```diff
@@ -885,6 +885,68 @@ describe('$mdDialog', function() {
       var sibling = angular.element(parent[0].querySelector('.sibling'));
       expect(sibling.attr('aria-hidden')).toBe('true');
     }));
+
+    it('should set tabindex of to sibling\'s tabbable nodes to -1', inject(function($mdDialog, $rootScope, $timeout) {
+
+      var template = '<md-dialog aria-label="Some Other Thing">Hello</md-dialog>';
+      var parent = angular.element('<a href="a">');
```

Comment:
@marcysutton I wanted to test that it didn't set tabindex of the parent to -1, so I just went with something tabbable. It could be pulled out into another test, or the test may not even be necessary.